tools/ci/appveyor/appveyor_build.cmd

```diff
@@ -21,7 +21,7 @@
 @echo off
 
 chcp 65001 > nul
-set BUILD_PREFIX=🐫реализация
+set BUILD_PREFIX=реализация
 set OCAMLROOT=%PROGRAMFILES%\Бактріан🐫
```

Comment:
It looks like this `🐫`  is also triggering Appveyor errors.

Reply:
There is a long-standing bug, but fixing that will require a Changes entry, so I'm going to retire both camels, merge this PR and then open a separate PR to fix the bug and restore one of them 🐪